api-docs/api-reference/methods/users/post-reset-pwd-v2.0.rst

```diff
@@ -0,0 +1,72 @@
+.. _post-reset-pwd-v2.0:
+
+Reset password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/reset-pwd
+
+Use the Reset Password operation to update users password.
```

Comment:
Use the reset password operation to update a user's password.  (You do need the parenthesis 😊 )